InvokePlaster.ps1

```diff
@@ -239,6 +239,20 @@ __________.__                   __
             $retval
         }
 
+        function GetGitConfigValue($name) {
```

Comment:
Oh yeah, can you move this helper function to the bottom of the file with the other helper functions?  Minor nit but would save me from having to do it.  :-)